src/compiler/transformer.ts

```diff
@@ -116,6 +117,7 @@ namespace ts {
             transformers.push(transformJsx);
         }
 
+        transformers.push(transformESNext);

```

Comment:
Do we plan on having a `--target esnext` to disable esnext transformations and emit as-is?